reactioncommerce/api-plugin-i18n
# feat: move files to plugin

This plugin replaces the internal `i18n` plugin. 

Once this PR is merged and this plugin is published to `npm`, there will be a related PR in `reaction` to swap out this plugin for the existing version.

## Testing

- see that the plugin is loaded upon startup in Reaction (use the `systemInformation` query)
- change a string and verify that you see the changes in Reaction Admin

Issue https://github.com/reactioncommerce/reaction/issues/6173

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
